lte/gateway/Makefile

```diff
@@ -148,7 +148,7 @@ build_python: stop ## Build Python environment
 build_common: format_common ## Build shared libraries
 	$(call run_cmake, $(C_BUILD)/magma_common, $(MAGMA_ROOT)/orc8r/gateway/c/common, $(COMMON_FLAGS))
 
-build_oai: format_core ## Build OAI
```

Comment:
Note that we should not explicitly build_common outside the cmake, this is what I tried to fix with the super build. Please revert this and if something is failing you don't have the right dependencies expressed in CMake.

Reply:
Removed.